plugins/processors/ec2tagger/ec2tagger.go

```diff
@@ -184,6 +182,51 @@ func (t *Tagger) Apply(in ...telegraf.Metric) []telegraf.Metric {
 	return in
 }
 
+// Shutdown currently does not get called, as telegraf does not have a cleanup hook for Filter plugins
+func (t *Tagger) Shutdown() {
```

Comment:
Moving all of the functions around in the file makes this a confusing diff so it'll take longer to review.

Reply:
Agree its take longer time and confusing. However, for better structure format (e.g following the [ecs_decorator](https://github.com/aws/amazon-cloudwatch-agent/blob/master/plugins/processors/ecsdecorator/ecsdecorator.go)), its worth to restructure it and better considerate for future reviewers.

Reply:
If that's the case, I will revert it back 👍